src/App.js

```diff
@@ -66,6 +67,12 @@ class App extends Component {
     this.setState({ search: '', movies: [], status: DEFAULT_STATUS, color: 'black'})
   }
 
+  clearHistory(e){
+    e.preventDefault()
+    const { history, search, status, movies } = this.state
```

Comment:
Is this line needed?

Reply:
no it doesn't seem to be. I took it out, thanks.